src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.container__review-stars {
+  display: flex;
+}
+
+.stars-star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+}
+
+.stars-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.container__review-stars :nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
Stars modifiers should work correctly
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/ee0ea6bb-59a6-492a-b77d-e08bcd15fc56)